app/models.py

```diff
@@ -30,6 +32,8 @@ class Service(db.Model):
                                nullable=False)
     data = db.Column(JSON)
 
+    status = db.Column(db.String, index=False, unique=False, nullable=False)
```

Comment:
Also, maybe it's worth making an integer status code column instead of the string and then adding an enum of status names in the model?